en/docs/learn/outbound-provisioning-with-salesforce.md

```diff
@@ -281,14 +281,14 @@ Provider.
     | LanguageLocaleKey                  | `               http://wso2.org/claims/dob              `                      |
     | LastName                           | `               http://wso2.org/claims/lastname              `                 |
     | LocaleSidKey                       | `               http://wso2.org/claims/primaryChallengeQuestion              ` |
-    | ProfileId                          | `               http://wso2.org/claims/role              `                     |
+    | ProfileId                          | `               None             `                                             |
```

Comment:
Do we need this None mapping

Reply:
please check the previous review's reply

Reply:
then shall we remove the mapping

Reply:
If we remove the mapping we can't add the ProfileId in Advanced Claim Configuration, so the API call will fail.

Reply:
This give the impression that `None` is the value to add here.
Therefore, shall we say the following:

```
| ProfileId                          | Empty                                           |
```

Reply:
yeah that's better. will do the changes. Thank you!

Reply:
@nilminiwso2 did the changes. please review